contrib/meson/meson/examples/meson.build

```diff
@@ -0,0 +1,49 @@
+# #############################################################################
+# Copyright (c) 2018-present    lzutao <taolzu(at)gmail.com>
+# All rights reserved.
+#
+# This source code is licensed under both the BSD-style license (found in the
+# LICENSE file in the root directory of this source tree) and the GPLv2 (found
+# in the COPYING file in the root directory of this source tree).
+# #############################################################################
+
+lz4_root_dir = '../../../..'
+
+#examples_c_args = ['-Wextra', '-Wundef', '-Wshadow', '-Wcast-align', '-Wstrict-prototypes']
```

Comment:
Was it forgotten?

Reply:
Sorry, no idea. I kept those files unmodified from upstream and will delete when they're part of the next release.
https://github.com/lz4/lz4/blob/dev/contrib/meson/meson/examples/meson.build#L12